### PR TITLE
[enterprise-4.11] CNV-22554_411: Removing hostname from cloud-init field

### PR DESCRIPTION
**Pull request by danielclowers**
**Version(s):**
4.11 only

**Issue:**
https://issues.redhat.com/browse/CNV-22554

**Link to docs preview:**

* https://57675--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-11-release-notes.html
* https://57675--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html#virt-cloud-init-fields-web_virt-create-vms
* https://57675--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html
* https://57675--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html

**QE review:**
- [x] QE has approved this change.

**Additional information:**
* Main PR https://github.com/openshift/openshift-docs/pull/56152 failed to merge for 4.11 and earlier
* Manually cherrypicking for [4.9](https://github.com/openshift/openshift-docs/pull/57830), [4.10](https://github.com/openshift/openshift-docs/pull/57801), and [4.11](https://github.com/openshift/openshift-docs/pull/57675)
* Cherrypicked from 59d1d495d01b1b07ed193029906d52ee257988d9

### CODE REVIEW

**Comment by jeana-redhat** [comment]
This needs the commits squashed to 1, then good to merge IMO

File: virt/virt-4-11-release-notes.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ The SVVP Certification applies to:
 //CNV-18319 UI can switch between BIOS and UEFI
 * You can set the xref:../virt/virtual_machines/virt-edit-vms.adoc#virt-editing-vm-web_virt-edit-vms[boot mode] of templates and virtual machines to *BIOS*, *UEFI*, or *UEFI (secure)* by using the web console.
 
-* You can now enable and disable the descheduler from the web console on the *Scheduling* tab of the xref:../virt/virtual_machines/virt-create-vms.adoc#virt-vm-fields-web_virt-create-vms[*VirtualMachine details*] page.
+* You can now enable and disable the descheduler from the web console on the *Scheduling* tab of the xref:../virt/virtual_machines/virt-create-vms.adoc[*VirtualMachine details*] page.
```
Comment (jeana-redhat) [inline virt/virt-4-11-release-notes.adoc:90]: Try this:
```suggestion
* You can now enable and disable the descheduler from the web console on the *Scheduling* tab of the xref:../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[*VirtualMachine details*] page.
```

**Comment by apinnick** [comment]
This needs to be reverted.

**Pull request merged**